.github/workflows/release.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
+        submodules: true
```

Comment:
Wait, I accidentally lied. C++ is the only one that needs it. Java gets its stuff from Maven. I was thinking the Java SDK was like the CRT for some reason.
I'll remove it from everything but C++ since it's the only one that needs it.